skymap_scanner/recos/__init__.py

```diff
@@ -62,6 +62,12 @@ def get_vertex_variations() -> List[I3Position]:
         """Returns a list of vectors referenced to the origin that will be used to generate the vertex position variation."""
         pass
 
+    @staticmethod
+    @abstractmethod
+    def get_use_online_ra_dec() -> bool:
+        """Returns a boll whether to use online_ra_dec to point the scan."""
+        pass
+
```

Comment:
Instead of defining this staticmethod in the base class (and setting `self.use_online_ra_dec = False` in most of the other derived classes), you could simplify this without any loss of info by detecting if `isint
SplineMPE_pointed`